api/config/settings.py

```diff
@@ -312,9 +312,9 @@
 MOBILE_ACTIVATION_TOKEN_HASH_ALGORITHM = 'sha256'
 
 # Test Settings
-TEST_PAYLOAD_PATH = str(API_DIR) + '/utils/test/'
```

Comment:
Was this affecting builds on travis?

Reply:
Yeah it is affecting the builds on travis since there is no module utils.test...tests are in the user folder. Thanks